cmd/livepeer/livepeer.go

```diff
@@ -270,8 +272,8 @@ func main() {
 		n.NodeType = core.TranscoderNode
 	} else if *broadcaster {
 		n.NodeType = core.BroadcasterNode
-	} else {
-		glog.Info("Node type not set with -broadcaster, -transcoder, -orchestrator or -redeemer")
+	} else if !*reward && !*initializeRound {
+		glog.Fatalf("No services enabled; must be one of -broadcaster, -transcoder, -orchestrator, -redeemer, -reward or -initializeRound")
```

Comment:
I don't think "one of" is entirely accurate anymore how about "must be at least one of" , not that the latter is entirely accurate because not all combinations are possible either.

Reply:
Fixed in 447190b